pkg/jobs/adopt.go

```diff
@@ -124,6 +130,16 @@ const (
 	canRunArgs = `(SELECT $3::TIMESTAMP AS ts, $4::FLOAT AS initial_delay, $5::FLOAT AS max_delay) args`
 	// NextRunClause calculates the next execution time of a job with exponential backoff delay, calculated
 	// using last_run and num_runs values.
```

Comment:
nit: This comment belongs to the query below I believe.

nit: Add comment for this newly added query?